tests/ui/large_enum_variant.stderr

```diff
@@ -31,6 +31,23 @@ help: consider boxing the large fields to reduce the total size of the enum
 LL |     ContainingLargeEnum(Box<LargeEnum>),
    |                         ~~~~~~~~~~~~~~
 
+error: large size difference between variants
+  --> $DIR/large_enum_variant.rs:39:5
+   |
+LL |     ContainingMoreThanOneField(i32, [i32; 8000], [i32; 9500]),
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ this variant is 70004 bytes
+   |
+note: and the second-largest variant is 8 bytes:
+  --> $DIR/large_enum_variant.rs:41:5
+   |
+LL |     StructLikeLittle { x: i32, y: i32 },
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+help: consider boxing the large fields to reduce the total size of the enum
+  --> $DIR/large_enum_variant.rs:39:5
+   |
+LL |     ContainingMoreThanOneField(i32, [i32; 8000], [i32; 9500]),
+   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Hmm this suggestion is missing `Box`es?

Reply:
> Hmm this suggestion is missing `Box`es?

Hi, thanks.  Yes, this situation should not trigger lint.

Reply:
I think it actually should trigger. Though it would take a little more work to detect which fields to box. A simpler fix here would be to just change the `span_help` to `help` (the span made me think it was a code suggestion when it's actually not).

Reply:
> I think it actually should trigger. Though it would take a little more work to detect which fields to box. A simpler fix here would be to just change the `span_help` to `help` (the span made me think it was a code suggestion when it's actually not).

Hi,  Should the result be like following two way？
1. 
Is ContainingMoreThanOneField(i32, [i32; 8000], [i32; 9500]),  -> 

enum LargeEnum3 {
    ContainingMoreThanOneField(i32, Box<[i32; 8000]>, Box<[i32; 9500]>),
    VoidVariant,
    StructLikeLittle { x: i32, y: i32 },
}

Or

2.

error: large size difference between variants
  --> $DIR/large_enum_variant.rs:39:5
   |
LL |     ContainingMoreThanOneField(i32, [i32; 8000], [i32; 9500]),
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ this variant is 70004 bytes
   |
note: and the second-largest variant is 8 bytes:
  --> $DIR/large_enum_variant.rs:41:5
   |
LL |     StructLikeLittle { x: i32, y: i32 },
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
help: consider boxing the large fields in ContainingMoreThanOneField to reduce the total size of the enum

Reply:
Either is good. The first is better.

Reply:
> Either is good. The first is better.

Done.